### PR TITLE
Added Multiple CVE and Misconfiguration Templates

### PR DESCRIPTION
**Pull request by Parimal-shaw**
Added Following Templates:

- CVE_2022_0150_WordPress_Accessibility_Helper_Lt_0_6_0_7_Cross_Site.bcheck
- CVE-2022-2460  VoipMonitor - Pre-Auth SQL Injection.bcheck
- CVE-2023-36346 POS Codekop v2.0 - Cross-site Scripting.bcheck
- CVE_2021_20114_TCExam_Gt_14_8_1_Sensitive_Information_Exposure.bcheck
- CVE_2021_20158_Trendnet_AC2600_TEW_827DRU_2_08B01_Admin_Password.bcheck
- CVE_2021_21816_D_Link_DIR_3040_1_13B03_Information_Disclosure.bcheck
- SAP Directory Listing.bcheck
- Xdebug_remote_code_execution_via_xdebug_remote_connect_back.bcheck
- Apache Tomcat Manager Path Normalization Panel.bcheck
- Cloudflare External Image Resizing Misconfiguration.bcheck
- Etcd Server - Unauthenticated Access.bcheck
- Kubernetes_Pods_API_Discovery_&_Remote_Code_Execution.bcheck
- Rails CRLF and XSS.bcheck

### CODE REVIEW

**Comment by olliewuk** [comment]
in `other/Apache Tomcat Manager Path Normalization Panel.bcheck` should the password be static as it is?

**Comment by Parimal-shaw** [comment]
Yes, it doesn't disclose current users password a user and change whatever they want the password to be .

**Comment by olliewuk** [comment]
> Yes, it doesn't disclose current users password a user can change whatever they want the password to be .

Should the check be modified in that case to not include a specific password?

**Comment by Parimal-shaw** [comment]
Not required , the current check cannot be changed in any other way the following checks required a password in static form or else the password will get rejected .

File: other/Apache Tomcat Manager Path Normalization Panel.bcheck [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+		"/bchecktest/..;/hostmanager/html"
+	
+
+given host then
+    send request called check:
+		method: "GET"
+		path: {potential_path}
+
+			if {check.response.status_code} is "403" 
+			and "username=\"tomcat\" password=\"s3cret\"" in {check.response.body}
```
Comment (olliewuk) [inline other/Apache Tomcat Manager Path Normalization Panel.bcheck:24]: if the user has changed the password will this line work? I have a concern you are checking this response specifically.

Comment (Parimal-shaw) [reply other/Apache Tomcat Manager Path Normalization Panel.bcheck:24]: No if the user has changed the password the following code will not work. As ther is no regex support in bcheck file we cannot mention a any regex value of there if regex support kindly let us know.

Comment (olliewuk) [reply other/Apache Tomcat Manager Path Normalization Panel.bcheck:24]: There is regex support in BCheck - see `{X} matches "[regex]"` in https://portswigger.net/burp/documentation/scanner/bchecks/bcheck-definition-reference

I think at this time this check is too fragile and that explicit password check should be replaced with a regex.

Comment (Parimal-shaw) [reply other/Apache Tomcat Manager Path Normalization Panel.bcheck:24]: Yes , sure will update that changes with regex

Comment (olliewuk) [reply other/Apache Tomcat Manager Path Normalization Panel.bcheck:24]: thank you

Comment (Parimal-shaw) [reply other/Apache Tomcat Manager Path Normalization Panel.bcheck:24]: Can you confirm rest of my bchecks.

Comment (Parimal-shaw) [reply other/Apache Tomcat Manager Path Normalization Panel.bcheck]: The changes have been made ,kindly check

**Review by olliewuk — APPROVED** [review]

**Comment by Parimal-shaw** [comment]
All the changes have been done ,suggested by you

**Review by PortSwiggerWiener — APPROVED** [review]
👍

**Pull request merged**